core/dbt/events/functions.py

```diff
@@ -344,3 +325,17 @@ def get_ts_rfc3339() -> str:
     ts = get_ts()
     ts_rfc3339 = ts.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
     return ts_rfc3339
+
+
+def add_to_event_history(event):
+    if flags.EVENT_BUFFER_SIZE == 0:
```

Comment:
We should be able to turn the event buffer off by setting to 0, which we couldn't do before.